### PR TITLE
Update ldap-integration.md

### PR DESCRIPTION
**Pull request by kerneljake**
general cleanup, sAMAccountName is commonly used in Active Directory (uid is OpenLDAP), REST or GUI works

### CODE REVIEW

**Comment by bmansheim** [comment]
@ushachar @RoeyPrat Guys, do you have any comments on this change?

File: content/rs/administering/security/ldap-integration.md [hunk 1]
```diff
@@ -41,17 +45,10 @@ the LDAP server.
 $ vi /tmp/saslauthd.conf
 ```
 
-Note: If you change the path for the conf file, be sure to change the
```
Comment (RoeyPrat) [inline content/rs/administering/security/ldap-integration.md]: why is this not necessary?

File: content/rs/administering/security/ldap-integration.md [hunk 2]
```diff
@@ -41,17 +45,10 @@ the LDAP server.
...
-system config file we configured in the previous step.
-
 You must specify the URIs for the LDAP servers you will be
 authenticating with. You can specify multiple LDAP servers by listing
 them separated by a space. 
 
-If you are using LDAP over SSL, then
```
Comment (RoeyPrat) [inline content/rs/administering/security/ldap-integration.md]: why is this not important?

File: content/rs/administering/security/ldap-integration.md [hunk 3]
```diff
@@ -62,7 +59,7 @@ ldap_search_base: ou=coolUsers,dc=company,dc=com
 
 # Specify a search filter. The value for the configuration option (%u)
 # should correspond to parameters (uid) specific for your installation.
```
Comment (RoeyPrat) [inline content/rs/administering/security/ldap-integration.md]: should the "uid" in the brackets not be sAMAccountName as well?

**Comment by bmansheim** [comment]
@ushachar I would like someone from R&D to approve this change. Any ideas?

File: content/rs/administering/security/ldap-integration.md (hunk 3 above)
Comment (bmansheim) [reply content/rs/administering/security/ldap-integration.md]: @danni-m Do you agree with Roey?

Comment (danni-m) [reply content/rs/administering/security/ldap-integration.md]: Makes sense `uid` in this context refers to the ldap_filter parameter.

Comment (bmansheim) [reply content/rs/administering/security/ldap-integration.md]: @danni-m Please approve the commit: https://github.com/RedisLabs/redislabs-docs/pull/181/commits/b0e558c638ccff13f58b4bc328cc44c29508f5e2

**Pull request merged**